src/components/InRequest/InRequest.tsx

```diff
@@ -1,654 +1,57 @@
-              Is Employee a New Air Force{" "}
-              {empType === EMPTYPES.Civilian ? "Civilian" : "Military"}?
-            </Label>
-            <Controller
-              name="isNewCivMil"
-              control={control}
-              rules={{
-                required: "Selection is required",
-              }}
-              render={({ field }) => (
-                <RadioGroup
-                  {...field}
-                  aria-describedby="isNewCivMilErr"
-                  id="newCivId"
-                >
-                  <Radio key={"yes"} value={"yes"} label="Yes" />
-                  <Radio key={"no"} value={"no"} label="No" />
-                </RadioGroup>
-              )}
-            />
-            {errors.isNewCivMil && (
-              <Text id="isNewCivMilErr" className={classes.errorText}>
-                {errors.isNewCivMil.message}
-              </Text>
-            )}
-            {isNewCivMil === "no" && (
-              <>
-                <Label htmlFor="prevOrgId">Previous Organization</Label>
-                <Controller
-                  name="prevOrg"
-                  control={control}
-                  rules={{
-                    required: "Previous Organization is required",
-                  }}
-                  render={({ field }) => (
-                    <Input
-                      {...field}
-                      aria-describedby="prevOrgErr"
-                      id="prevOrgId"
-                    />
-                  )}
-                />
-                {errors.prevOrg && (
-                  <Text id="prevOrgErr" className={classes.errorText}>
-                    {errors.prevOrg.message}
-                  </Text>
-                )}
-              </>
-            )}
-          </>
-        )}
-        {(empType === EMPTYPES.Civilian || empType === EMPTYPES.Military) && (
-          <>
-            <Label htmlFor="newToBaseAndCenterId">
-              Is Employee new to WPAFB and AFLCMC?
-            </Label>
-            <Controller
-              name="isNewToBaseAndCenter"
-              control={control}
-              rules={{
-                required: "Selection is required",
-              }}
-              render={({ field }) => (
-                <RadioGroup
-                  {...field}
-                  aria-describedby="isNewToBaseAndCenterErr"
-                  id="newToBaseAndCenterId"
-                >
-                  <Radio key={"yes"} value={"yes"} label="Yes" />
-                  <Radio key={"no"} value={"no"} label="No" />
-                </RadioGroup>
-              )}
-            />
-            {errors.isNewToBaseAndCenter && (
-              <Text id="isNewToBaseAndCenterErr" className={classes.errorText}>
-                {errors.isNewToBaseAndCenter.message}
-              </Text>
-            )}
-          </>
-        )}
-
-        {empType === EMPTYPES.Contractor && (
+  if (!request.isLoading && request.data) {
```

Comment:
Not sure what this is providing?  If request.isLoading then line 17 in React will return "Loading" and shouldn't load this fragment yet.  If it is no longer is loading, then we should have data, right? If we don't have data then we should display an error message about being unable to load the Request.  Check out topic 4 from one of the react-query maintainers blog.  It outlines a possible strategy for handling data errors etc -- https://tanstack.com/query/v4/docs/community/tkdodos-blog#4-status-checks-in-react-query